core/src/test/java/org/opensearch/sql/planner/logical/LogicalPlanNodeVisitorTest.java

```diff
@@ -44,9 +43,7 @@
 import org.opensearch.sql.storage.write.TableWriteBuilder;
 import org.opensearch.sql.storage.write.TableWriteOperator;
 
-/**
- * Todo. Temporary added for UT coverage, Will be removed.
- */
+/** Todo. Temporary added for UT coverage, Will be removed. */
```

Comment:
The most permanent thing is a temporary one. Probably, update the comment:
```suggestion
/** Added for UT coverage. */
```

Reply:
Addressed in `integ/sl_GoogleJavaFormat5`